src/components/Modal/Modal.tsx

```diff
@@ -46,18 +48,20 @@ const Modal = ({
   keyboard = true,
   scrollable,
   centered,
+  onHide,
   ...rest
 }: ModalProps) => (
   <ModalRB
     size={size}
-    fullScreen={fullScreen}
+    fullscreen={fullScreen}
```

Comment:
We should keep the variable case consistent so might be worth changing them all the the proper case